stylesheets/partials/layouts/agency-homepage.scss

```diff
@@ -0,0 +1,138 @@
+/*ADD TO ANIMAL SERVICES FOLDER*/
```

Comment:
@mxsnyder agreed, should be broken into components if possible, as well as removing the red-inside overrides once that is cleaned up. this was just a means of giving sam a quick css file that was source controled until we make it sassier.